vue2-vuetify/src/layouts/ArrayLayoutRenderer.vue

```diff
@@ -303,11 +334,30 @@ const controlRenderer = defineComponent({
     const suggestToDelete = ref<null | number>(null);
     // indicate to our child renderers that we are increasing the "nested" level
     useNested('array');
+    const t = useTranslator();
+    const defaultAddNewItemLabelText = 'Add new';
+    const defaultAddNewItemAriaLabelText = 'Add to';
```

Comment:
Sounds good. Do you mean something like a messages file? eg

```typescript
{
   'jsonforms.arraylayout.newItem': 'Add new'
}
```

And if so do you have a preference of .ts or .json file?

Reply:
I don't have a specific preference. A json file could potentially be easier to copy for translation purposes so I would probably go that way.

Reply:
Re. the add new item button being different - I have a requirement to change it's style for my implementation, but didn't want to force that on all users. Is this a situation where the template compiler mentioned in https://github.com/eclipsesource/jsonforms-vuetify-renderers/pull/40 would be applicable? So it would be possible to inject a custom button, without needing to replicate the entire array layout renderer?

Reply:
Hmm I don't think it's a perfect match. The template renderer really does runtime compilation so arbitrary templates within UI Schemas can be shown.

In your case you just want to adjust the array button in general, so the advantage of the template renderer, it's flexibility, is not really used. However you will still have all the downsides, e.g. larger bundle size, slower rendering and respecifying the button in all UI Schemas (or configs) in all forms you show.

I think what we should use instead is a slot-based system so the renderer can easily be reused as a custom renderer without the need to copy any code.

Reply:
Of course - slots solves this problem neatly.